lib/util/AuthConfig.js

```diff
@@ -111,9 +120,64 @@ class AuthConfig extends Config {
             this.loginKey = undefined;
             this.refreshToken = undefined;
         }
+
+        var tempJSON = this._getAuthJSON();
+        
+
+        if(tempJSON == undefined) {
+            tempJSON = {};
+        }
+
+        if(Object.values(this._json)[0].isDefault) {
+            for (var account in tempJSON) {
+                if(tempJSON[account].isDefault){
+                    delete tempJSON[account].isDefault;
+                }
+            }
+        }
+        
+        const accountID = Object.keys(this._json)[0];
+        tempJSON[accountID] = Object.values(this._json)[0];
+
+        this._json = tempJSON;
+
         return super.save();
     }
 
+    _init() {
+        this._exists = false;
+        this._isValid = true;
+        this._json = {};
+    }
+
+    _getAuthJSON() {
+        const content = Utils.readFileSync(this._fileName);
+        if (content !== null) {
+            try {
+                return JSON.parse(content);
```

Comment:
Since no human should be touching these files it's probably not worth it, but just an FYI, I'm a big fan of [relaxed JSON](https://www.npmjs.com/package/relaxed-json) so that you can add comments, etc.